.changeset/silly-shoes-agree.md

```diff
@@ -2,4 +2,4 @@
 "@blitzjs/generator": patch
 ---
 
-Remove `-cookie-prefix` appended to the cookiePrefix string
+Remove `-cookie-prefix` appended to the cookiePrefix string in the generator template. This will also fix auth and CSRF issues for people upgrading.
```

Comment:
I would also mention what kind of auth and CSRF issues because if we have more than one (or will have), someone reading the changelog may expect ALL of them to be fixed (or at least they won't be sure which ones are solved).
```suggestion
Remove `-cookie-prefix` appended to the `cookiePrefix` config property in the new app template. It will also fix auth and CSRF issues for users upgrading from a legacy framework.
```